strings/charoccurrence.go

```diff
@@ -0,0 +1,20 @@
+// charoccurrence.go
+// description: Chars occurrence in the sentence
+// details: An algorithm to count chars in string
```

Comment:
```suggestion
// details: An algorithm which counts the number
// of times a character occurred in a string.
```

Reply:
- [x] fixed

Reply:
This is not fixed - look over what you have added.